docs/uml/sequenceDiagrams/miscellaneous/ArchitectureSequenceDiagram.puml

```diff
@@ -10,36 +10,37 @@ Participant ":UI" as UI #YellowGreen
 Participant ":Parser" as Parser #Gold
 Participant ":Storage" as Storage #IndianRed
```

Comment:
The colour for the Storage component has been changed. Perhaps you could include the `Style.puml` file and use the constant `COLOR_STORAGE` instead? Can refer to #290 if you need a guide.

Reply:
Ok, I will change it later. Since the color is the same so I didn't remove it.